### DISCUSSION TITLE
Welcome to Carbon Discussions!

### DISCUSSION THREAD

**Discussion by joshblack** [carbon-design-system/carbon#7417] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jnm2377** [comment]
Thanks for starting this discussion @joshblack 🎉 


👋🏽 👋🏽 👋🏽  _Hi to all in our community!_ 👋🏽 👋🏽 👋🏽 

I am **Josefina Mancilla (she/her)**. I am a front end developer who helps maintain Carbon Design System. I've been working on Carbon since late 2018 and am so happy to be part of this awesome community! Feel free to connect with me on Twitter [@brosefina_nm](https://twitter.com/brosefina_nm) and check-out my portfolio www.josefina.codes, which is actually still a WIP and let's be honest, might be for a while but will hopefully/eventually include a blog lol 

Also, I am bilingual, and love to connect with our Latin/Spanish-speaking Carbon community if anyone is more comfortable speaking in Spanish. 😄

**Comment by dakahn** [comment]
Hello! I'm D.A. (he/him). I'm a frontend developer and web accessibility advocate on Carbon Components. Feel free to tag me in an accessibility related questions/concerns. 🏄🏽
